Lib/test/support/__init__.py

```diff
@@ -1667,7 +1667,8 @@ def missing_compiler_executable(cmd_names=[]):
     missing.
 
     """
-    from distutils import ccompiler, sysconfig, spawn, errors
+    from distutils import ccompiler, spawn, errors
```

Comment:
This will probably be better continuing to use distutils.sysconfig for now, but please add a `# TODO (PEP 632): alternate check without using distutils` comment.

Reply:
I'm gonna revert the commit where this is implemented. We'll whether it impacts tests somehow.